src/identity/contactMapping.ts

```diff
@@ -207,6 +211,34 @@ export function* fetchAddressesAndValidateSaga({
   }
 }
 
+export function* fetchAddressVerificationSaga({ address }: FetchAddressVerificationAction) {
+  try {
+    const verifiedAddresses = yield* select(verifiedAddressesSelector)
+    if (verifiedAddresses.includes(address)) {
+      ValoraAnalytics.track(IdentityEvents.address_lookup_skip)
```

Comment:
would this ever be true? seems like the action is only dispatched if address isn't in the verifiedAddresses list

Reply:
No, at least, not with how this is being called now. Not strictly necessary to have this, but it's a simple check and a bit more defensive.

Reply:
Should the analytics event here then if it's not ever going to be called in the current usage of this saga? Can we drop it or move it to the SelectRecipient screen?

Reply:
I think we're probably fine to drop it, I'll get rid of it.